sandbox/trim-low-abund.py

```diff
@@ -36,27 +40,41 @@
 MAX_FALSE_POSITIVE_RATE = 0.8
 
 
-def output_single(read, trim_at=None):
-    name = read.name
-    sequence = read.sequence
-
-    accuracy = None
+def trim_record(read, trim_at):
+    new_read = _screed_record_dict()
```

Comment:
I like being explicit about what is being transferred here, so -0 on using deepcopy.

That having been said, IMO we need either or both of:

* a public way to create a new screed record;
* a trimming function to do what is used here.

I will add both to issues.